marc_to_solr/lib/traject_config.rb

```diff
@@ -1131,11 +1131,7 @@
 
 to_field 'holdings_1display' do |record, accumulator|
   all_holdings = process_holdings(record)
-  if all_holdings == {}
-    logger.error "#{record['001']} - Missing holdings"
-  else
-    accumulator[0] = all_holdings.to_json.to_s
-  end
+  accumulator[0] = all_holdings.to_json.to_s unless all_holdings.empty?
 end
 
```

Comment:
I think it is useful to have this info. In the current production catalog there are not a lot but there are some. I reviewed the error log with @mzelesky a while ago and we didnt add a note to remove this from the log. https://github.com/pulibrary/bibdata/issues/858 There is a google document with an older log. I would also ask @mzelesky again.

Reply:
If it's helpful I will keep it but downgrade it to info or debug. but it would be nice to hear a scenario where we've used it. With the alma indexes there are really a ton of these entries.

Reply:
I have used the log in the past to review the index and this is why it was added. When we first reviewed the log there was not an alma index to compare the results. I understand that there are a lot of records in alma with no holdings and makes it difficult to review the log and this came up now with the alma qa index. I think we can remove it for now unless there is a different opinion from @mzelesky .

Reply:
Yes, I think it's ok to remove for now. Eventually, it would be useful to know which bib records have no physical holdings or electronic portfolios, but we can take that as it comes.

Reply:
@mzelesky per you request i opened https://github.com/pulibrary/bibdata/issues/1230. Thank you.